OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -378,10 +378,10 @@ public AssetBrowserLogic(Widget widget, Action onExit, ModData modData, WorldRen
 			}
 
 			var assetBrowserModData = modData.Manifest.Get<AssetBrowser>();
-			allowedSpriteExtensions = assetBrowserModData.SpriteExtensions;
-			allowedModelExtensions = assetBrowserModData.ModelExtensions;
-			allowedAudioExtensions = assetBrowserModData.AudioExtensions;
-			allowedVideoExtensions = assetBrowserModData.VideoExtensions;
+			allowedSpriteExtensions = assetBrowserModData.SpriteExtensions.Select(x => x.ToLowerInvariant()).ToArray();
+			allowedModelExtensions = assetBrowserModData.ModelExtensions.Select(x => x.ToLowerInvariant()).ToArray();
```

Comment:
Why ToArray? They are going to be immediately converted and used as IEnumerables anyway. And at the end converted ToArray regradless

Reply:
They are also used individually later.

Reply:
Oh I see. But then still, if you are assigning IEnumerable to Array doesn't it automatically call ToArray?

Reply:
No, it tells you you can't assign it.